Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_ObjectAutoGrab.cs

```diff
@@ -5,36 +5,38 @@
 
     public class VRTK_ObjectAutoGrab : MonoBehaviour
     {
-        public GameObject objectToGrab;
+        public VRTK_InteractableObject objectToGrab;
         public bool cloneGrabbedObject;
 
         private VRTK_InteractGrab controller;
 
         private IEnumerator Start()
         {
             controller = GetComponent<VRTK_InteractGrab>();
-
             if (!controller)
             {
                 Debug.LogError("The VRTK_InteractGrab script is required to be attached to the controller along with this script.");
+                yield break;
```

Comment:
This is not how we should handle this IMO. This is a setup issue you should fix once. The old state was plain wrong since when the code continued to run (or you simply did not have errorpause on like most have) it would work on null objects. Also at least for me the error did not even appear in the console, it was simply hidden by Unity for some reason. This solution should be much cleaner and informative IMO.